editoast/migrations/2024-01-17-140850_rollingstock-add-supported-signaling-systems/up.sql

```diff
@@ -0,0 +1,12 @@
+-- Your SQL goes here
+
+ALTER TABLE rolling_stock
+ADD supported_signaling_systems jsonb NOT NULL DEFAULT ('["BAPR", "BAL"]');
+
+UPDATE rolling_stock
+SET supported_signaling_systems='["BAPR", "BAL", "TVM300", "TVM430"]'
+WHERE metadata->>'detail' LIKE '%TGV%' OR name LIKE '%TGV%';
+
+UPDATE rolling_stock
+SET supported_signaling_systems='["BAPR", "BAL", "TVM430"]'
+WHERE name='1TGVDUPL';
```

Comment:
These don't support TVM300?

Reply:
that’s the only one who supports 430 but not 300, from the data we have